packages/api/tests/controllers/nomination.test.js

```diff
@@ -52,6 +52,31 @@ describe('GET Nomination Endpoint', () => {
   });
 });
 
+describe('GET ALL Nomination Endpoint', () => {
+  let nomination;
+
+  beforeAll(() => {
+    nomination = db.Nomination.build(nominationData);
+    return nomination.save();
+  });
+
+  afterAll(() => {
+    nomination.destroy();
+  });
+
+  it('returns a 200 when nominations exist', async () => {
+    const res = await request(app).get(`/nominations`);
+    expect(res.statusCode).toBe(200);
+    expect(res.body.length).toEqual(1);
+  });
+
+  it('return a 404 when no nomination exist', async () => {
+    await db.Nomination.destroy({ where: {} });
```

Comment:
This makes the tests interdependent. Maybe we should just create the nomination in the first test on 67-71 (and destroying all nominations after each test), instead of creating it globally and then deleting it in this test